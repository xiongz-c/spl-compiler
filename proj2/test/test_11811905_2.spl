int test;
char test;

struct test {};

int test() {}

struct test {};

int test() {}

int test_11811806_2() {
    int b = (2.0);

    return 0;
}