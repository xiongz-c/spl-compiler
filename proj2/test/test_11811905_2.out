Error type 3 at Line 2: redefine variable: test
Error type 15 at Line 8: redefine struct: test
Error type 4 at Line 10: redefine function: test
Error type 5 at Line 13: unmatching type on both sides of assignment
